### PR TITLE
styles

### PR DESCRIPTION
**Pull request by drorwolmer**
image

### CODE REVIEW

File: app/components/ChangeMusicInput/ChangeMusicInput.tsx [hunk 1]
```diff
@@ -28,10 +28,7 @@ const ChangeMusicInput = memo(() => {
   };
 
   return (
-    <div
-      className="form-group bg-dark"
-      style={{ padding: '5px 10px', margin: '0px' }}
-    >
+    <div className="form-group" style={{ padding: '5px 10px', margin: '0px' }}>
```
Comment (smuk3c) [inline app/components/ChangeMusicInput/ChangeMusicInput.tsx:31]: Why not define it inside ChangeMusicInout.css directly?

File: app/main.dev.ts [hunk 2]
```diff
@@ -32,21 +32,21 @@ import { machineId, machineIdSync } from 'node-machine-id';
...
+//     'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789';
+//   const charactersLength = characters.length;
+//   // eslint-disable-next-line no-plusplus
+//   for (let i = 0; i < length; i++) {
+//     result += characters.charAt(Math.floor(Math.random() * charactersLength));
+//   }
+//   return result;
+// }
+//
+// const machineID = makeid(6);
```
Comment (smuk3c) [inline app/main.dev.ts:49]: Can it be removed?

File: package.json [hunk 3]
```diff
@@ -275,6 +276,7 @@
     "menubar": "^9.0.1",
     "node-abi": "^2.19.1",
     "node-machine-id": "^1.1.12",
+    "popper.js": "^1.16.1",
```
Comment (smuk3c) [inline package.json:279]: Why popper is used for?

**Review by smuk3c — APPROVED** [review]

File: package.json (hunk 3 above)
Comment (drorwolmer) [reply package.json:279]: a depepndency of bootstrap...

File: app/components/ChangeMusicInput/ChangeMusicInput.tsx (hunk 1 above)
Comment (drorwolmer) [reply app/components/ChangeMusicInput/ChangeMusicInput.tsx:31]: we will, we need to refactor all the inlince css

**Pull request merged**